daemon/graphdriver/windows/windows.go

```diff
@@ -369,6 +380,7 @@ func (d *Driver) DiffSize(id, parent string) (size int64, err error) {
 	return archive.ChangesSize(layerFs, changes), nil
 }
 
+// RestoreCustomImages allows to add a custom image to the tag and graph store
```

Comment:
Should be something like:
RestoreCustomImages adds any auto-detected OS specific images to the tag and graph store.